msteams-platform/concepts/device-capabilities/people-picker-control-capability.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Integrate People Picker control capability
```

Comment:
People Picker control capability is a bit confusing since "control" and "capability" both convey similar meanings. So, let's stick with "People Picker capability" in all places, including the page URL. This also aligns with our existing naming convention of using "capability" at end of the features.